.circleci/Dockerfile

```diff
@@ -14,6 +14,7 @@ FROM    circleci/golang:1.10rc1
 ARG     K8S_VER=v1.9.2
 ARG     ETCD_VER=v3.2.15
 ARG     MINIKUBE_VER=v0.25.0
+ARG     HELM_VER=v2.8.0
```

Comment:
Not sure if it matters, however, we specify Helm 2.7.2 throughout the documentation as a minimum pin.  I personally think until 1.0 latest stable releases are the easiest path, but have read on the ML that 3 y-stream releases are under support until 1.y.z.